src/modules/visits/controllers/visits.controller.ts

```diff
@@ -226,7 +226,10 @@ export class VisitsController {
 		@SessionUser() user: SessionUserEntity
 	): Promise<Visit | null> {
 		// Check if the user is a CP admin or a Kiosk user for the requested space
```

Comment:
REQ: this comment line doesn't seem to be accurate anymore please adjust

Reply:
Yes it should be moved to line 262
and this comment should read:
// MEEMOO_ADMIN has access to all the visitor spaces